src/localization/locales/locale.en.ts

```diff
@@ -1,3 +1,18 @@
-const en = {};
+const en = {
+    sections: {
+        intro: 'Introduction',
+        api: 'API REST',
+        expressjs: 'Express JS',
+        mongoosejs: 'Mongoose',
+    },
+    common: {
+        components: {
+            lastReviewed: 'Last reviewed on ',
+        },
+    },
+    titles: {
+        backendStandars: 'Backend standars',
```

Comment:
@PVillasana The title can be Backend Standars? Using Capital letter in the firs letter of the second word? (Pascal case more and less)

Reply:
@gonzalo-plaza You mean the variable name or the title value? if the last one is the case, im not using any specific casing since it is just a name.

Reply:
@PVillasana Nice, I resolve de conversation.